components/addFolderFlyout/index.test.jsx

```diff
@@ -19,6 +19,7 @@
  * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
+import React from "react";
```

Comment:
These should be auto-imported; was the linter complaining?

Reply:
It seems to be Jest itself, giving me `ReferenceError: React is not defined`.